sdk/tests/js/tests/gl-object-get-calls.js

```diff
@@ -964,6 +964,16 @@ if (contextVersion > 1) {
 	    return gl.getActiveUniforms(program, uniformIndices, pname);
         }
     );
+
+    debug("");
+    debug("Test getUniformBlockIndex");
+    var program = wtu.loadUniformBlockProgram(gl);
+    gl.linkProgram(program);
+    shouldBeTrue('gl.getProgramParameter(program, gl.LINK_STATUS)');
+    shouldBeNonNull('gl.getUniformBlockIndex(program, "Transform")');
+    var blockIndex = gl.getUniformBlockIndex(program, "Transform");
+    gl.uniformBlockBinding(program, blockIndex, 1);
+    shouldBe('gl.getUniformBlockIndex(program, "Transform")', '1');
```

Comment:
Are you trying to verify the binding?  Then you should use getActiveUniformBlockParameter() instead.  Besides, it is another get function and should not be put here.